modules/swagger-jaxrs2/src/test/java/io/swagger/jaxrs2/annotations/operations/AnnotatedOperationMethodTests.java

```diff
@@ -0,0 +1,321 @@
+        String extractedYAML = openApiYAML.substring(start, end);
+        String expectedYAML = "---\n" +
+                "openapi: \"3.0.0-rc2\"\n" +
+                "paths:\n" +
+                "  /pet/{petId}:\n" +
+                "    get:\n" +
+                "      summary: \"Find pet by ID\"\n" +
+                "      description: \"Returns a pet when 0 < ID <= 10.  ID > 10 or nonintegers will\\\n" +
+                "        \\ simulate API error conditions\"\n" +
+                "      parameters:\n" +
+                "      - name: \"petId\"\n" +
+                "        in: \"path\"\n" +
+                "        description: \"ID of pet that needs to be fetched\"\n" +
+                "        required: true\n" +
+                "        explode: false\n" +
+                "        schema:\n" +
+                "          type: \"integer\"\n" +
+                "          format: \"int64\"\n" +
+                "      responses:\n" +
+                "        default:\n" +
+                "          description: \"The pet\"\n" +
+                "          content:\n" +
+                "            '*/*':\n" +
+                "              schema:\n" +
+                "                type: \"object\"\n" +
+                "                properties:\n" +
+                "                  id:\n" +
+                "                    type: \"integer\"\n" +
+                "                    format: \"int64\"\n" +
+                "                  name:\n" +
+                "                    type: \"string\"\n" +
+                "        400:\n" +
+                "          description: \"Invalid ID supplied\"\n" +
+                "        404:\n" +
+                "          description: \"Pet not found\"\n" +
+                "  /pet:\n" +
+                "    put:\n" +
+                "      summary: \"Update an existing pet\"\n" +
+                "      parameters:\n" +
+                "      - description: \"Pet object that needs to be added to the store\"\n" +
+                "        required: true\n" +
+                "        explode: false\n" +
+                "        schema:\n" +
+                "          $ref: \"#/components/schemas/Pet\"\n" +
+                "      responses:\n" +
+                "        400:\n" +
+                "          description: \"Invalid ID supplied\"\n" +
+                "        404:\n" +
+                "          description: \"Pet not found\"\n" +
+                "        405:\n" +
+                "          description: \"Validation exception\"\n" +
+                "    post:\n" +
+                "      summary: \"Add a new pet to the store\"\n" +
+                "      parameters:\n" +
+                "      - description: \"Pet object that needs to be added to the store\"\n" +
+                "        required: true\n" +
+                "        explode: false\n" +
+                "        schema:\n" +
```

Comment:
`parameter` object should _never_ be missing an `in` attribute.  In this case, the parameter body must be a `requestBody` rather than a `parameter`.